Update azure-arc-enable-cluster.md

Greedygre opened the pull request:
Fix the following script errors：
1. Remove unused parameter assignments, this parameter will be assigned repeatedly later in the article
2. $wORKSPACE_NAME use uppercase
3. Removing spaces that would cause an error in az aks create script
4. Set location when creating connected environment
5. Fix variable myConnectedEnvironment,myContainerApp  assignment
6. Fix container app image path

Greedygre committed:
Update azure-arc-enable-cluster.md

Fix the following script errors：
1. Remove unused parameter assignments, this parameter will be assigned repeatedly later in the article.
2. $wORKSPACE_NAME use uppercase
3. Removing spaces that would cause an error in az aks create script

Greedygre committed:
Update azure-arc-enable-cluster.md

Set location when create connected-env

Greedygre committed:
Update azure-arc-create-container-app.md

1. ContainerApp name must consist of lower-case alphanumeric characters or '-', start with an alphabetic character
2. Fix variable myConnectedEnvironment assignment

Greedygre committed:
Update azure-arc-create-container-app.md

fix

Greedygre committed:
Update azure-arc-create-container-app.md

fix script for get connected environment resource Id.

howang-ms commented on articles/container-apps/azure-arc-create-container-app.md:
````diff
@@ -75,7 +75,9 @@ Now that you have the custom location ID, you can query for the connected enviro
 
 ```azure-interactive
-myConnectedEnvironment = az containerapp connected-env list --custom-location customLocationId -o tsv --query '[].id'
+myResourceGroup="my-container-apps-resource-group"
````
should we move this to the line 46

Greedygre replied:
Yes, moved to line 46.

Greedygre committed:
Update azure-arc-create-container-app.md

Fix variable myConnectedEnvironment,myContainerApp assignment

Greedygre committed:
Update azure-arc-enable-cluster.md

When installing extension, configuration-settings set clusterName, rather than CLUSTER_NAME

LaylaLiu-gmail commented on articles/container-apps/azure-arc-enable-cluster.md:
```diff
@@ -316,7 +314,7 @@ A [Log Analytics workspace](../azure-monitor/logs/quick-create-workspace.md) pro
         --configuration-settings "Microsoft.CustomLocation.ServiceAccount=default" `
         --configuration-settings "appsNamespace=${NAMESPACE}" `
-        --configuration-settings "CLUSTER_NAME=${CONNECTED_ENVIRONMENT_NAME}" `
+        --configuration-settings "clusterName=${CONNECTED_ENVIRONMENT_NAME}" `
```
also need to update line 336

Greedygre replied:
done

Greedygre committed:
Update azure-arc-enable-cluster.md

Change clusterName description.

cebundy commented on articles/container-apps/azure-arc-create-container-app.md:
```diff
@@ -112,7 +114,7 @@ let StartTime = ago(72h);
 ContainerAppsConsoleLogs_CL
 | where TimeGenerated between (StartTime .. EndTime)
-| where AppName_s =~ "myContainerApp"
+| where AppName_s =~ "my-container-app"
```
Shouldn't this now be ...
```suggestion
| where AppName_s =~ $myContainerApp
```

Greedygre replied:
This is executed in the log analysis workspace, not in bash, so need to enter the actual value directly.

Greedygre replied:
Hi, I abandon this pr, my colleague merged my changes in this pr: https://github.com/MicrosoftDocs/azure-docs/pull/102998
Many thanks for your review!
